parm/hrrr_postcntrl_gtg.xml

```diff
@@ -0,0 +1,1222 @@
+<?xml version="1.0"?>
```

Comment:
@YaliMao-NOAA Since this control file could be used for the upcoming DAFS implementation, I suggest to remove duplicated ops. HRRR fields and only keep gtg and new aviation fields.